.github/workflows/ci.yaml

```diff
@@ -90,6 +90,26 @@ jobs:
           path: ${{github.workspace}}/clar2wasm/src/standard/standard.wasm
           if-no-files-found: error
 
+  build:
+    name: Build binaries
+    runs-on: ubuntu-latest
+    needs: [build-test-artifacts]
+    steps:
+      - name: Checkout PR
+        uses: actions/checkout@v4
+
+      - name: Use Rust stable
+        uses: dtolnay/rust-toolchain@stable
+
+      - name: Download archive
```

Comment:
This step can be removed since the test artifacts are not need.

Reply:
done